src/index.html

```diff
@@ -1,12 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--fri calendar--month-length--31">
+
+      <div class="calendar__container">
```

Comment:
For this purpose better use mix.
This way you could avoid unnecessary wrappers and set all site blocks to `page` elements - convenient and right for bigger projects.
 More details in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)